### ISSUE TITLE
Milvus failed to start when the mysql account has no table create permission

### ISSUE THREAD

**Issue by mayunjie** [milvus-io/milvus#6651]
#### Is your feature request related to a problem? Please describe:
The current milvus uses the create table if not exist statement when it is started. This statement requires the account to have the permission to create tables, but for many companies, we always do not have the prd environment database table create permission.

#### Describe the solution you'd like:
conifg in server_config.yaml whether need create table default

#### Describe alternatives you've considered:
first check the table is exist or not, skip create if there is already has one table when startup

**Comment by yhmo** [comment]
The #6670 make a fix.

When milvus start, it use "SHOW TABLES LIKE xxx" to check whether the meta schema exist. If the schema didn't exist, no need to call "CREATE TABLE IF NOT EXIST".

**Comment by yhmo** [comment]
Test steps:
1. start mysql server
`docker run -p 3306:3306 -e MYSQL_ROOT_PASSWORD=123456 -d mysql:5.7`

2. use root account to login the mysql server, create a database named "milvus"
 `mysql -h127.0.0.1 -uroot -p123456`
`CREATE DATABASE milvus;`

3. create a new user, assign these privileges: insert, delete, update, select
```
create user 'tom' identified by '123456';
grant insert on milvus.* to 'tom';
grant delete on milvus.* to 'tom';
grant update on milvus.* to 'tom';
grant select on milvus.* to 'tom';
```
4. set meta uri in server_config.yaml
```
general:
  timezone: UTC+8
  meta_uri: mysql://tom:123456@127.0.0.1:3306/milvus
```
5. start milvus server
6. test sdk api such as: create_collection/drop_collection/insert/create_index/flush/search

**Comment by yhmo** [comment]
Test steps:
1. start mysql server
`docker run -p 3306:3306 -e MYSQL_ROOT_PASSWORD=123456 -d mysql:5.7`

2. use root account to login the mysql server, create a database named "milvus"
 `mysql -h127.0.0.1 -uroot -p123456`
`CREATE DATABASE milvus;`

3. manually create milvus meta schema by root account
```
CREATE TABLE IF NOT EXISTS Tables (id BIGINT PRIMARY KEY AUTO_INCREMENT,table_id VARCHAR(255) UNIQUE NOT NULL,state INT NOT NULL,dimension SMALLINT NOT NULL,created_on BIGINT NOT NULL,flag BIGINT DEFAULT 0 NOT NULL,index_file_size BIGINT DEFAULT 1024 NOT NULL,engine_type INT DEFAULT 1 NOT NULL,index_params VARCHAR(512) NOT NULL,metric_type INT DEFAULT 1 NOT NULL,owner_table VARCHAR(255) NOT NULL,partition_tag VARCHAR(255) NOT NULL,version VARCHAR(64) DEFAULT '1.1.1',flush_lsn BIGINT DEFAULT 0 NOT NULL);

CREATE TABLE IF NOT EXISTS TableFiles (id BIGINT PRIMARY KEY AUTO_INCREMENT,table_id VARCHAR(255) NOT NULL,segment_id VARCHAR(255) NOT NULL,engine_type INT DEFAULT 1 NOT NULL,file_id VARCHAR(255) NOT NULL,file_type INT DEFAULT 0 NOT NULL,file_size BIGINT DEFAULT 0 NOT NULL,row_count BIGINT DEFAULT 0 NOT NULL,updated_time BIGINT NOT NULL,created_on BIGINT NOT NULL,date INT DEFAULT -1 NOT NULL,flush_lsn BIGINT DEFAULT 0 NOT NULL);

CREATE TABLE IF NOT EXISTS Environment (global_lsn BIGINT NOT NULL);

CREATE TABLE IF NOT EXISTS Fields (collection_id VARCHAR(255) NOT NULL,field_name VARCHAR(255) NOT NULL,field_type INT DEFAULT 0 NOT NULL,field_params VARCHAR(255) NOT NULL);

```

4. create a new user, assign these privileges: insert, delete, update, select
```
create user 'tom' identified by '123456';
grant insert on milvus.* to 'tom';
grant delete on milvus.* to 'tom';
grant update on milvus.* to 'tom';
grant select on milvus.* to 'tom';
```
5. set meta uri in server_config.yaml
```
general:
  timezone: UTC+8
  meta_uri: mysql://tom:123456@127.0.0.1:3306/milvus
```
6. start milvus server
7. test sdk api such as: create_collection/drop_collection/insert/create_index/flush/search

**Comment by yhmo** [comment]
@mayunjie 

The #5330 was fixed on branch 1.1, the official docker image of 1.1.2 is not yet published, you can download a release-candidate image of 1.1.2:
```
docker pull milvusdb/milvus-poc:1.1.2-cpu-rc
docker pull milvusdb/milvus-poc:1.1.2-gpu-rc
```
